sway/input/seat.c

```diff
@@ -381,7 +381,8 @@ void seat_set_focus_warp(struct sway_seat *seat,
 		if (last_ws) {
 			ipc_event_workspace(last_ws, container, "focus");
 			if (last_ws->children->length == 0) {
-				container_workspace_destroy(last_ws);
+				output_damage_whole(last_ws->parent->sway_output);
```

Comment:
It's not this function's responsibility to damage outputs

Reply:
where do you put it?

Reply:
Inside the destructor?